pymc3/stats.py

```diff
@@ -115,7 +115,8 @@ def dic(trace, model=None):
 
     Returns
     -------
-    `float` representing the deviance information criterion of the model and trace
+    z : float
```

Comment:
Including the type here is very helpful -- thanks!  

The name of the return is optional -- in this case I'd argue it is more confusing to include, since the returned variable is not called `z`.  I'm not very familiar, but if this is a standard name in the literature, feel free to ignore!

The format should be just

    float
        The deviance information criterion of the model and trace

Reply:
copied from here https://github.com/scikit-learn/scikit-learn/blob/ef5cb84a805efbe4bb06516670a9b8c690992bd7/sklearn/metrics/regression.py#L488